Attach DisplayService is needed in Kit

<!--- Provide a brief summary of the issue -->
After #720, loading the mobile template Basic Screen fails:
```
Caused by: java.lang.ClassNotFoundException: com.gluonhq.attach.display.DisplayService
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
 ```

The DisplayService dependency should be build time (no runtime) and the display module should be added to Kit.

--- a/kit/pom.xml
+++ b/kit/pom.xml
@@ -71,7 +71,6 @@
             <artifactId>display</artifactId>
             <version>${gluon.attach.version}</version>
             <classifier>desktop</classifier>
-            <scope>runtime</scope>
         </dependency>
 
         <!-- Rest API -->

--- a/kit/src/main/java/module-info.java
+++ b/kit/src/main/java/module-info.java
@@ -39,6 +39,7 @@ module com.gluonhq.scenebuilder.kit {
     requires transitive javafx.web;
 
     requires com.gluonhq.charm.glisten;
+    requires com.gluonhq.attach.display;
     requires static javax.json.api;
     requires transitive static java.prefs;
 

--- a/pom.xml
+++ b/pom.xml
@@ -22,8 +22,8 @@
         <maven.compiler.release>${java.version}</maven.compiler.release>
         <javafx.version>23</javafx.version>
         <aether.version>1.1.0</aether.version>
-        <charm.glisten.version>6.2.2</charm.glisten.version>
-        <gluon.attach.version>4.0.19</gluon.attach.version>
+        <charm.glisten.version>6.2.3</charm.glisten.version>
+        <gluon.attach.version>4.0.21</gluon.attach.version>
         <junit.jupiter.version>5.10.0</junit.jupiter.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.github.repository>gluonhq/scenebuilder</project.github.repository>

